scrapers/GGW.yml

```diff
@@ -0,0 +1,26 @@
+name: "Girls Gone Wild"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - girlsgonewild.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[contains(@class,"watch__title")]/text()
+      Details: //div[contains(@class,"text-mob-more")]/text()
```

Comment:
Change to `Details: //div[contains(@class,"text-mob-more")]`. Currect selector cuts off details after certain point.